docs/standard/io/snippets/pipes/pipes-from-streams/ConsoleApp.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
folder for the snippet should be the name of the article so I believe the csproj and related files should be in this folder: `docs/standard/io/snippets/pipelines/` so `pipes/pipes-from-streams` is not needed.

Reply:
I'll simplify that, didn't realize we had specific naming convention for snippet folders. I'll be sure to do that moving forward.